### ISSUE TITLE
Questions for repeating examples in instructions

### ISSUE THREAD

**Issue by yangshenglan** [pycalphad/pycalphad#142]
Hi, Richard.
Examples1:
`db = Database('alfe_sei.TDB')
my_phases = ['LIQUID', 'B2_BCC']
eq = equilibrium(db, ['AL', 'FE', 'VA'], my_phases, {v.X('AL'): 0.25, v.T: (300, 2000, 50), v.P: 101325},
                 output=['heat_capacity', 'degree_of_ordering'])
print(eq)`
Examples2:
`db_alni = Database('NI_AL_DUPIN_2001.TDB')
phases = ['LIQUID', 'FCC_L12']
eq_alni = equilibrium(db_alni, ['AL', 'NI', 'VA'], phases, {v.X('AL'): 0.10, v.T: (300, 2500, 20), v.P: 101325},
                      output='degree_of_ordering')
print(eq_alni)`

I have the following questions in repeating above two examples.
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2

I don't know the reason for the question. I hope you help me to solve it. Look forward to your reply!

**Comment by bocklund** [comment]
This seems like a compiler issue. How did you install pycalphad? With conda or pip?

**Comment by yangshenglan** [comment]
I installed pycalphad with pip.

**Comment by bocklund** [comment]
Yes, it is probably a compiler issue then. We highly recommend installing pycalphad on Windows by using Anaconda, which should work without any issues. See the [pycalphad installation instructions here](https://pycalphad.org/docs/latest/INSTALLING.html).

**Comment by yangshenglan** [comment]
Hi Bocklund. I also tried to install pycalphad with conda, but I installed pycalphad with 'conda config --add channels conda-forge' and than 'conda install pycalphad' according to the pycalphad installation instructions and  always had the following question. Is poor Network about it? What is a good way to do this? @bocklund 
CondaError: CondaHTTPError: HTTP None None for url <https://conda.anaconda.org/c
onda-forge/win-64/openssl-1.0.2l-vc14_0.tar.bz2>
Elapsed: None

**Comment by yangshenglan** [comment]
Hi Bocklund. I also tried to install pycalphad with conda, but I installed pycalphad with 'conda config --add channels conda-forge' and than 'conda install pycalphad' according to the pycalphad installation instructions and  always had the following question. Is poor Network about it? What is a good way to do this?
CondaError: CondaHTTPError: HTTP None None for url <https://conda.anaconda.org/c
onda-forge/win-64/openssl-1.0.2l-vc14_0.tar.bz2>
Elapsed: None
